packages/components/src/common/CloseButton/CloseButton.stories.tsx

```diff
@@ -29,11 +23,23 @@ Default.args = {
 export const SmallerSize = Template.bind(null);
 SmallerSize.args = {
   ...defaultArgs,
-  size: "28px",
+  size: "1rem",
```

Comment:
I made reduced this value just so it's more obvious that it's smaller than the other stories.